### PR TITLE
Update Index “index”

### PR DESCRIPTION
**Pull request by vanessa-w**
Automatically generated by Netlify CMS

### CODE REVIEW

File: index.markdown [hunk 1]
```diff
@@ -2218,71 +2213,40 @@ Feel free to make a copy and alter it according to your own needs.
...
-  join)
-* \#identity-gender-minority (contact a member of
-  @identity-gender-minority-admins to join)
+#### People and HR Team
 
-Anyone can create a new one by creating the private channel, creating a user
-group with at least one volunteer (or asking a Slack admin to do it for you),
-and adding it to this list. It's probably a good idea to share that you've
-created it in #general.
+The People and HR team is here to support you during your time at dxw. We provide administrative, wellbeing, learning and development, line management and professional support and guidance.
```
[the remark this answers is not in the document]
Comment (vanessa-w) [reply index.markdown:2218]: Good spot - I'll change to line manager instead, hopefully that will be more clear

[the remark this answers is not in the document]
Comment (vanessa-w) [reply index.markdown:2218]: Not sure how to make that change at the moment though

[the remark this answers is not in the document]
Comment (james) [reply index.markdown:2218]: Just go back to the CMS page you did the edit in, make the change and hit 'save', it should update this pull request automatically. I should put that in the guide!

[the remark this answers is not in the document]
Comment (vanessa-w) [reply index.markdown]: Done - thanks

**Review by erbridge — APPROVED** [review]

File: index.markdown [hunk 2]
```diff
@@ -2354,6 +2318,70 @@ anything to him, don’t hesitate to get in touch. Dave's diary is open and you
 can see it on Google Calendar. If you would like to schedule some time with him,
 please feel free to book something.
 
+#### Learning and Development
```
Comment (erbridge) [inline index.markdown:2321]:
```suggestion
#### Learning and development
```

**Pull request merged**